xrayvision/Visibility.py

```diff
@@ -60,8 +118,53 @@ def to_map(self, outmap):
         """
         return Transform.idft(outmap, self.uv, self.vis)
 
+    def to_map_v2(self, outmap, center=None, pixel_size=None):
+        """
+        Parameters
+        ----------
+        outmap: np.array
+            An empty array with the same dimensions as
+            the output image should have
+        center: array-like
+            Where should be the origin in the Fourier-transformation
+        pixel_size: array-like
+            What is the size of one pixel in the given direction (x, y)
+
+        Returns
+        -------
+
+        """
+        if not center:
+            center = self.xyoffset
+        if not pixel_size:
+            pixel_size = self.pixel_size
+
+        return Visibility.idft_map(self.vis, outmap, self.uv, center, pixel_size)
+
+    def to_sunpy_map(self, dim):
```

Comment:
Maybe use a named keyword such as with reasonable defaults something like `size=(33, 33)`

Reply:
Nice work, only a few minor things I think might need to be changed